includes/Discovery.php

```diff
@@ -262,22 +298,64 @@ public function print_twitter_metadata() {
 		if ( ! $enable_twitter_metadata ) {
 			return;
 		}
-		?>
-		<meta name="twitter:card" content="summary_large_image" />
-		<?php
+		$metadata = $this->get_twitter_metadata();
 
-		if ( ! has_post_thumbnail() ) {
-			return;
+		foreach ( $metadata as $name => $value ) {
+			printf( '<meta name="%s" content="%s" />', $name, $value );  // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
That's dangerous, because output is not escaped when I add additional stuff through the `web_stories_story_twitter_metadata` filter.

Should use `esc_attr()` here (for both name **and** value), and remove `esc_attr()` from `get_twitter_metadata`.

-> basically escaping as late as possible

Reply:
Fair point. Fixed in https://github.com/google/web-stories-wp/pull/5919/commits/11ab8119541426348958a72ec412460d410e3c0d